.github/workflows/update-epm.yml

```diff
@@ -0,0 +1,28 @@
+name: Update OBO Extended Prefix Map
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * 1"
+jobs:
+  update_context:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        
+      - name: Work around permission issue
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+
+      - name: Update OBO context
+        run: |
+          wget "https://raw.githubusercontent.com/biopragmatics/bioregistry/main/exports/contexts/obo.epm.json" -O obo.epm.json
```

Comment:
If you want to commit the prefix map to the repository (which is a good idea for build reproducibility, no argument here), please don’t write it in the top-level directory (which is cluttered enough already).

I’d suggest a `resources/` subdirectory or similar.

Reply:
Done!